docs/_themes/clusterhq/static/css/docs.css

```diff
@@ -275,10 +275,6 @@ p {
     display: none;
 }
 
-.admonition a {
```

Comment:
Please separate this and put it against https://clusterhq.atlassian.net/browse/FLOC-1292